src/encryption.ts

```diff
@@ -89,7 +95,11 @@ export function encryptOnWrite(
           console.warn(warnings.whereClause(operation, path))
         }
         try {
-          const cipherText = encryptStringSync(clearText, keys.encryptionKey)
+          const cipherText =
+            encryptFn !== undefined
+              ? encryptFn(clearText)
```

Comment:
Sounds good. Can you add some documentation about the feature in the README please? Emphasizing that key management is left to the user's appreciation would be good, since the options to pass keys in code are still available (or we could disallow them if a custom cipher is used, via a TS discriminating union, up to you).